dashboard/src/app/index.module.js

```diff
@@ -96,8 +96,51 @@ initModule.factory('AuthInterceptor', ($window, $cookies, $q, $location, $log) =
   };
 });
 
-initModule.config(['$routeProvider', '$locationProvider', '$httpProvider', ($routeProvider, $locationProvider, $httpProvider) => {
+initModule.factory('AddMachineTokenToUrlInterceptor', ($injector, $q) => {
```

Comment:
because it's only used in development mode 
https://github.com/codenvy/codenvy/blob/master/dashboard/src/app/index.module.js#L103
then there is no minified version of this in dev